DependencyInjection/Configuration.php

```diff
@@ -81,7 +81,7 @@ public function getConfigTreeBuilder()
                             ->booleanNode('strict')->defaultTrue()->end()
                             ->booleanNode('escaping')->defaultTrue()->end()
                             ->booleanNode('line_breaks')->defaultTrue()->end()
-                            ->integerNode('max_newlines')->defaultValue(3)->end()
+                            ->scalarNode('max_newlines')->defaultValue(3)->end()
```

Comment:
@alquerci i think we should drop support for symfony versions, that are not maintained anymore.